cmd/terramate/cli/project.go

```diff
@@ -225,25 +225,38 @@ func (p project) checkDefaultRemote() error {
 	)
 }
 
-func (p *project) checkLocalDefaultIsUpdated() error {
+func (p *project) checkRemoteDefaultBranchIsReachable() error {
 	gitcfg := p.gitcfg()
+
+	remoteDesc := fmt.Sprintf("remote(%s/%s)", gitcfg.DefaultRemote, gitcfg.DefaultBranch)
+
+	logger := log.With().
+		Str("HEAD", p.headCommit()).
+		Str("default_branch", remoteDesc).
+		Str("default_hash", p.remoteDefaultCommit()).
+		Logger()
+
+	outOfDateErr := errors.E(
+		ErrCurrentHeadIsOutOfDate,
+		"Please merge the latest changes from %s into this branch",
+		remoteDesc,
```

Comment:
let's rephrase and also remove inline context.

```suggestion
		"Please update the current branch with the latest changes from the default branch.",
```